README.md

```diff
@@ -1 +1,22 @@
-# scheduled-codeship-build
\ No newline at end of file
+# scheduled-codeship-build
+
+This lambda is intended to trigger a project build on Codeship.
+It makes use of the *codeship-go* API client.
+
+## Required Environment Variables (for the lambda config)
+
+* CS_ORGANIZATION // the Codeship organization
+* CS_PASSWORD // the Codeship password
+* CS_USERNAME // the Codeship username for authentication
+* CS_PROJECT_UUID // the uuid of the target Codeship project
+* CS_BUILD_REFERENCE // the desired reference for the build, e.g. "20.04" or "develop"
+(note: "heads/" will be prepended to this value, if it is not already there)
+
+## Testing against Codeship
+
+Add the real environment variable entries to your local.env file.
```

Comment:
Will you be adding a `example.local.env` file?

Reply:
I don't think so. There is already and aws.env.example and the docker-compose.yml file has a comment about referring to that for examples.